src/murmur/Meta.h

```diff
@@ -58,6 +59,8 @@ class MetaParams {
 	bool bCertRequired;
 	bool bForceExternalAuth;
 
+	QList<VoiceProtocolType> allowedVoiceProtocolTypes;
```

Comment:
```suggestion
	QVector<VoiceProtocolType> allowedVoiceProtocolTypes;
```
I have read once that `QVector` should usually be preferred.

Any particular reason why this could not be a `std::vector`?

Reply:
Nah. I like `std::vector` but I just try to keep my code look like the old code. But I can change to `std::vector`. Your idea?

Reply:
I think using a `std::vector` would be better (in the context of us wanting to eventually ditch Qt for as many parts as possible)